### DISCUSSION TITLE
Disable elastalert rule

### DISCUSSION THREAD

**Discussion by ishukeshri2712** [jertel/elastalert2#1497] [Q&A]
I disabled the elastalert rule will it stop querying the documents from elasticsearch cluster?

**Comment by jertel** [comment]
Yes

**Comment by ishukeshri2712** [comment]
@jertel I have another doubt suppose we have an elastalert rule which query over all documents and we have put timestamp of last 5 minutes then it will query fresh documents which are pushed to elasticsearch ?

**Comment by jertel** [comment]
This is unrelated to the discussion topic. Please create a new discussion.
